src/CMakeLists.txt

```diff
@@ -84,11 +84,14 @@ elseif(WIN32) # incl. win64
                   -DNDEBUG)
 endif()
 if(${CMAKE_CXX_COMPILER_ID} STREQUAL "GNU")
-  add_definitions(-DNTA_COMPILER_GNU)
+  add_definitions(-DNTA_COMPILER_GNU
+                  -DNTA_SERIALIZATION_ON)
```

Comment:
Lol, yes I read that change wrong. Will submit a change with this removed. Also need to add a stream assert check and a unit test for it..